[ffmpeg] failed when use ffmpeg by cmake

**Describe the bug**
Failed to use ffmpeg by call `find_package(FFMPEG REQUIRED)` in CMakeLists.txt

**Environment**
- OS: Windows

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install ffmpeg[drawtext,opencl,x264,x265,webp,vpx] --triplet=v142-x64-static-mt
2. use ffmpeg in self project by cmake

**Expected behavior**
Success to run `cmake --build`

**Failure logs**
```
c:\Self\ffmpeg-test>cmake --preset x64-static-debug
Preset CMake variables:

  CMAKE_BUILD_TYPE="Debug"
  CMAKE_SYSTEM_VERSION="10.0.22621.0"
  CMAKE_TOOLCHAIN_FILE="c:\Self\vcpkg/scripts/buildsystems/vcpkg.cmake"
  IS_DEBUG_BUILD="ON"
  IS_DYNAMIC_BUILD="OFF"
  OUTPUT_PATH="C:/Self/ffmpeg-test/output/x64-static-debug"
  VCPKG_TARGET_TRIPLET="v142-x64-static-mt"

-- The C compiler identification is MSVC 19.29.30158.0
-- The CXX compiler identification is MSVC 19.29.30158.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.29.30133/bin/HostX64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.29.30133/bin/HostX64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- VCPKG_TARGET_TRIPLET: v142-x64-static-mt
-- VCPKG_DLL_DIR:
-- IS_DEBUG_BUILD: ON
-- IS_DYNAMIC_BUILD: OFF
-- CMAKE_CONFIGURATION_TYPES: Debug
-- OUTPUT_PATH: C:/Self/ffmpeg-test/output/x64-static-debug
CMake Error at C:/Self/vcpkg/installed/v142-x64-static-mt/share/ffmpeg/FindFFMPEG.cmake:73 (find_library):
  Could not find FFMPEG_DEPENDENCY_usp10_RELEASE using the following names:
  usp10
Call Stack (most recent call first):
  C:/Self/vcpkg/installed/v142-x64-static-mt/share/ffmpeg/FindFFMPEG.cmake:147 (append_dependencies)
  C:/Self/vcpkg/installed/v142-x64-static-mt/share/ffmpeg/vcpkg-cmake-wrapper.cmake:27 (_find_package)
  C:/Self/vcpkg/scripts/buildsystems/vcpkg.cmake:813 (include)
  CMakeLists.txt:54 (find_package)


-- Configuring incomplete, errors occurred!
```

The attachment is my modification. After the modification, it returned to normal.

[fix-use-ffmpeg-failed.patch](https://github.com/user-attachments/files/18879101/fix-use-ffmpeg-failed.patch)

--- a/ports/ffmpeg/FindFFMPEG.cmake.in
+++ b/ports/ffmpeg/FindFFMPEG.cmake.in
@@ -55,7 +55,7 @@ function(append_dependencies out)
     endif()
     set(pass_through
         ${CMAKE_CXX_IMPLICIT_LINK_LIBRARIES}
-        advapi32 bcrypt crypt32 gdi32 mfuuid ole32 oleaut32 psapi secur32 shlwapi strmiids user32 uuid vfw32 ws2_32
+        advapi32 bcrypt crypt32 gdi32 mfuuid ole32 oleaut32 psapi secur32 shlwapi strmiids user32 uuid vfw32 ws2_32 usp10 cfgmgr32 rpcrt4
         -pthread -pthreads pthread atomic m
     )
     cmake_policy(SET CMP0057 NEW)

--- a/ports/ffmpeg/vcpkg.json
+++ b/ports/ffmpeg/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "ffmpeg",
   "version": "7.1",
-  "port-version": 2,
+  "port-version": 3,
   "description": [
     "A library to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.",
     "FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created. It supports the most obscure ancient formats up to the cutting edge. No matter if they were designed by some standards committee, the community or a corporation. It is also highly portable: FFmpeg compiles, runs, and passes our testing infrastructure FATE across Linux, Mac OS X, Microsoft Windows, the BSDs, Solaris, etc. under a wide variety of build environments, machine architectures, and configurations."

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2754,7 +2754,7 @@
     },
     "ffmpeg": {
       "baseline": "7.1",
-      "port-version": 2
+      "port-version": 3
     },
     "ffnvcodec": {
       "baseline": "12.2.72.0",

--- a/versions/f-/ffmpeg.json
+++ b/versions/f-/ffmpeg.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "50a315a91c855d9daf05fca0a09bdd467e946972",
+      "version": "7.1",
+      "port-version": 3
+    },
+    {
       "git-tree": "88c48d9d8743bb6641072c4aa50e1da155c2eb50",
       "version": "7.1",
       "port-version": 2

